website/src/components/QuoteCarousel.tsx

```diff
@@ -0,0 +1,60 @@
+import { useCarousel } from '../hooks/carousel'
+import classNames from 'classnames'
+import React, { FunctionComponent, ReactFragment, ReactNode } from 'react'
+import { BlockquoteWithLogo } from './Blockquote'
+import ArrowLeftIcon from 'mdi-react/ArrowLeftIcon'
+import ArrowRightIcon from 'mdi-react/ArrowRightIcon'
+
+interface Blockquote {
+    quote: string
+    header?: string
+    by?: string | ReactFragment
+    logoHref?: string
+    logoImage?: string
+    linkText?: string
+    link?: string
+}
+
+interface QuoteCarouselProps {
+    items: ReactNode[]
+    currentItem?: ReactNode
+    previousItem?: ReactNode
+    currentItemIndex?: number
+    autoAdvance?: boolean
+    className?: string
+}
+
+export const QuoteCarousel: FunctionComponent<QuoteCarouselProps> = ({ items, autoAdvance, className }) => {
+    const carouselHook = useCarousel(items, autoAdvance ?? false)
+    const carouselItems = carouselHook.carouselItems.items as Blockquote[]
+    const carouselStyles = 'd-flex flex-lg-row flex-column justify-content-center text-center'
+
+    return (
+        <div className={className ? classNames(carouselStyles, className) : classNames(carouselStyles)}>
+            <div className="d-lg-flex d-none align-items-center">
```

Comment:
The [designs for this page](https://www.figma.com/file/aA1cNEm2JWtBSuijZyxROs/Fixing-vulnerabilities-page?node-id=177%3A2) call for this behavior (`d-lg-flex`/ `d-none`), which means that there's not a way to manually advance between panels on mobile (autoadvance will work). This seems ... odd to me?

Reply:
Sweet, yeah, I'm gonna add that behavior then. I will also add a note to the top-level PR description that we've decided to add that functionality.